test/sharness/t0109-gateway-web-_redirects.sh

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env bash
+
+test_description="Test HTTP Gateway _redirects support"
+
+. lib/test-lib.sh
+
+test_init_ipfs
+test_launch_ipfs_daemon
+
+## ============================================================================
+## Test _redirects file support
+## ============================================================================
+
+# Directory tree crafted to test _redirects file support
+test_expect_success "Add the _redirects file test directory" '
+  mkdir -p testredirect/ &&
+  echo "my index" > testredirect/index.html &&
+  echo "my one" > testredirect/one.html &&
+  echo "my two" > testredirect/two.html &&
+  echo "my 404" > testredirect/404.html &&
+  mkdir testredirect/redirected-splat &&
+  echo "redirected splat one" > testredirect/redirected-splat/one.html &&
+  echo "/redirect-one /one.html" > testredirect/_redirects &&
+  echo "/301-redirect-one /one.html 301" >> testredirect/_redirects &&
+  echo "/302-redirect-two /two.html 302" >> testredirect/_redirects &&
+  echo "/200-index /index.html 200" >> testredirect/_redirects &&
+  echo "/posts/:year/:month/:day/:title /articles/:year/:month/:day/:title 301" >> testredirect/_redirects &&
+  echo "/splat/:splat /redirected-splat/:splat 301" >> testredirect/_redirects &&
+  echo "/en/* /404.html 404" >> testredirect/_redirects &&
+  echo "/* /index.html 200" >> testredirect/_redirects &&
+  REDIRECTS_DIR_CID=$(ipfs add -Qr --cid-version 1 testredirect)
+'
```

Comment:
Mind creating a CAR that has the test dir, and using it instead of constructing it on the fly? 
We have some prior art in `test/sharness/t0118-gateway-car/carv1-basic.car`

That way we could use its root CID in the RFC  under "Test Fixtures" section :)

Reply:
> If you add me as Admin to https://github.com/justincjohnson/go-ipfs-redirects i'll move it to ipfs org + ensure you still have your permissions.

Sounds good.  I sent you an invite.  Once you accept, I'll make you an admin.

Reply:
I have a draft PR at https://github.com/ipfs/specs/pull/290.  I'm waiting until the gateway specs and RFC PRs merge before I make mine official.